koiki/tests/test_create_delivery.py

```diff
@@ -71,7 +72,38 @@ def setUp(self):
             ]
         }
 
+    @responses.activate
     def test_body(self):
+        responses.add(responses.GET, 'https://wcfmmp_testing_host/wp-json/wcfmmp/v1/settings/id/5',
```

Comment:
Maybe we could add this url "https://wcfmmp_testing_host/wp-json/wcfmmp/v1" in the tests/__init.py__ also for DRY

Reply:
yes, good catch!

Reply:
Sure, since i saw 
koiki.wcfmmp_api_base = 'https://wcfmmp_testing_host'

in your lazona.coop/lazona_connector/koiki/tests/__init__.py  I guessed you could profit of it.